packages/projection-typeorm/src/entity/Block.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { Cardano } from '@cardano-sdk/core';
+import { Column, Entity, Index, PrimaryColumn, RelationOptions } from 'typeorm';
+
+@Entity({ name: 'block' })
+export class BlockEntity {
+  @PrimaryColumn({ length: 64, type: 'char' })
```

Comment:
Using a 64 character field as primary key doesn't sound to be a good idea: all other entries referring to this table will need to spend a lot of space to refer to this table.
A 32 bytes integer field (up to 2 billions blocks) as primary key for this table would be much more efficient.

Reply:
Note that it can sometimes make queries more complicated, if you already have block hash like [here](https://github.com/input-output-hk/cardano-js-sdk/pull/616/commits/1eee6b82e4fe3c5e3ff16ea545f2e41c2b7f4313#diff-fb271f632f8478f3ec6b07bbc5218c7bdbd118313fe542f37be2e91acff55208R183)